google-tests/CMakeLists.txt

```diff
@@ -6,58 +6,37 @@ FetchContent_Declare(
 	GIT_TAG main
 )
 
-include_directories(soltrace_example_tests hello_world ../coretrace ../coretrace/strace ../app/deploy/samples)
+include_directories(soltrace_example_tests csv_tools ../coretrace ../coretrace/strace ../app/deploy/samples)
 
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 FetchContent_MakeAvailable(googletest)
 
 
-file(GLOB SAMPLE_FILES ./samples/*.stinput)
-file(GLOB SOLTRACE_EXAMPLE_TESTS soltrace_example_tests/st_sim_run_test.cpp soltrace_example_tests/st_mtrand_test.cpp)
+file(GLOB SAMPLE_FILES ./samples/*.stinput ./samples/*.mon)
+file(GLOB GROUND_FILES ./ground_results/*.csv)
+file(GLOB SOLTRACE_EXAMPLE_TESTS soltrace_example_tests/*.cpp)
+file(GLOB CSV_TOOL csv_tools/*.cpp csv_tools/*.h)
 
 set(STRACE_SRC ../coretrace/strace)
-
 set(CORETRACE ../coretrace/stapi.cpp)
 set(STRACE ../coretrace/strace/strace.cpp)
 set(ENV{SOLTRACE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}")
 
-# Copying files to build directory on building seems to be a common practice in cmake.
-# I wanted to use configure_file to copy these files, but this command seems to have issues with relative paths 
-#(such as the ones created when globbing the sample files together)
-
-#configure_file(${SAMPLE_FILES} ${CMAKE_CURRENT_BINARY_DIR} COPYONLY)
-
 
 file(COPY ${SAMPLE_FILES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
why should we have two "samples" directories in the repo? Why not just copy from the deploy directory?